test/integration/features/cmd-addons.feature

```diff
@@ -18,7 +18,7 @@ Feature: Addons command and its subcommands
   Scenario: Installing add-on from a folder
   Note: working directory when executing Minishift commands is: /test/integration.
```

Comment:
I think the working directory would be ```out/test-run```

Reply:
@amitkrout Good catch, going to update now.